src/test/java/org/stellar/sdk/Sep10ChallengeTest.java

```diff
@@ -84,8 +84,8 @@ public void testNewChallengeRejectsMuxedClientAccount() throws InvalidSep10Chall
           timeBounds
       );
       fail();
-    } catch (FormatException e) {
-      assertEquals("Version byte is invalid", e.getMessage());
+    } catch (InvalidSep10ChallengeException e) {
```

Comment:
Isn't this an odd exception for this scenario?

Reply:
a sep10 challenge can only be generated using non-muxed accounts. Hence `InvalidSep10ChallengeException` is thrown when trying to validate a challenge token which has a muxed account